test/integration/command_test.exs

```diff
@@ -110,7 +110,22 @@ defmodule Integration.CommandTest do
   end
 
   test "running a command in a pipeline with nil output", %{user: user} do
-    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --field="b" | echo $a))
+    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --path=b | echo $a))
     assert response["data"]["response"] == "1"
   end
+
+  test "reading the path to filter a certain path", %{user: user} do
+    response = send_message(user, ~s(@bot: seed '[{"foo":{"bar":{"baz":"stuff"}}}, {"foo": {"bar":{"baz":"me"}}}]' | operable:filter --path="foo.bar"))
+    assert response["data"]["response"] == "{\n  \"bar\": {\n    \"baz\": \"stuff\"\n  }\n}\n{\n  \"bar\": {\n    \"baz\": \"me\"\n  }\n}"
```

Comment:
This isn't the correct behavior. `filter --path=...` does not _extract_ the value found at the end of the path, it allows the whole object through. If the object has nothing at that path, it does not make it through the filter.